static-site/src/pages/groups.jsx

```diff
@@ -4,23 +4,39 @@ import { SmallSpacer, HugeSpacer, FlexCenter } from "../layouts/generalComponent
 import * as splashStyles from "../components/splash/styles";
 import { fetchAndParseJSON } from "../util/datasetsHelpers";
 import DatasetSelect from "../components/Datasets/dataset-select";
+import ListResources from "../components/ListResources/index";
 import { GroupCards } from "../components/splash/groupCards";
 import GenericPage from "../layouts/generic-page";
 import { UserContext } from "../layouts/userDataWrapper";
 import { DataFetchErrorParagraph } from "../components/splash/errorMessages";
 import { groupsTitle, GroupsAbstract } from "../../data/SiteConfig";
 
-const datasetColumns = ({isNarrative}) => [
-  {
-    name: isNarrative ? "Narrative" : "Dataset",
-    value: (d) => d.request.replace("/groups/", "").replace(/\//g, " / "),
-    url: (d) => d.url
-  },
-  {
-    name: "Group Name",
-    value: (d) => d.request.split("/")[2],
-    url: (d) => `/groups/${d.request.split("/")[2]}`
-  }
+const sourceId = "charon/groups";
+async function parserCallBackFxn(response) {
+    const datasets = (await response.json()).datasets;
+
+    // Use "" instead of fake date because downstream components can
+    // conditionalize display choices based on `== ""` or `!== ""`
+    const pathVersions = Object.assign(
+        ...datasets.map((ds) => ({
+            [ds.request.replace(/^\/groups\//, "")]: [""]
+        })),
+    );
```

Comment:
This seems like a hack around fragile logic under `versioned=false` that requires `PathVersions` values to be `string[]` which doesn't mesh well with `/groups`. Ideally it can be `undefined`:

```ts
export interface PathVersions {
  [name: string]: string[] | undefined
}
```

This would be a larger refactor which also necessitates attributes like `Resource.lastUpdated` to be optional, but maybe worth doing now.

Reply:
I didn't like the empty string approach, but using `undefined` or `nil` didn't feel great either, and changing all the other code to deal with the value-side _not_ being an array felt like a *lot* of work.

Reply:
> using `undefined` or `nil` didn't feel great either

`undefined` sounds right to me... e.g. "what are the dates associated with this path?"

1. `undefined` – there are no dates!
2. there are no dates, but here's a single date with the value `""`

> changing all the other code to deal with the value-side _not_ being an array felt like a _lot_ of work.

It might be nontrivial, but I would consider it in scope of making ListResources re-usable outside of `/pathogens` and `/staging` pages (same reasoning as https://github.com/nextstrain/nextstrain.org/pull/904#discussion_r1631608768). Otherwise, ListResources will remain tailored for those two "first-class" pages, with pages such as `/groups` considered "second-class" and relying on workarounds to match assumptions made for the first-class pages.

Reply:
> Otherwise, ListResources will remain tailored for those two "first-class" pages, with pages such as /groups considered "second-class" and relying on workarounds to match assumptions made for the first-class pages.

This is a compelling point for me. In the short-term future we will probably have:
* 1 page with versions (/pathogens)
* 1 page with last-updated date (/staging)
* lots of pages with no dates (/groups, `/groups/<NAME>`, community splash pages)

And that's not including [a files listing page](https://github.com/nextstrain/nextstrain.org/pull/828) which would itself require larger changes. 

For what it's worth, I'd use an empty array to represent no dates.

Reply:
> For what it's worth, I'd use an empty array to represent no dates.

To be explicit: will there _ever_ be a resource that has *multiple* undated datasets? 

If the answer to that is "no", then sure, an empty array will work to indicate "no date".

If that answer to that is "...maybe", then I think an array with one `undefined` value per dataset is a more future proof approach.

Reply:
If we had such data then the internal structure would have to change anyway because the UI would need a way to distinguish between them, ultimately by surfacing different URLs for each.

The difference between `[]` and `[undefined]` in JS is subtle and I don't know if I'd write code that relied on that subtlety.

Reply:
> If we had such data then the internal structure would have to change anyway because the UI would need a way to distinguish between them, ultimately by surfacing different URLs for each.

okay, good to know. 

> The difference between `[]` and `[undefined]` in JS is subtle and I don't know if I'd write code that relied on that subtlety.

the main reason i would reach for `[undefined]` instead of `[]` is because all the existing code expects to iterate over a list, and `[undefined]` _can_ be iterated over. 

i will try to see how ugly an empty list will make this; i expect a couple parts will get more funky than they currently are — we'll see.

Reply:
`[undefined]` is fine by me

Reply:
> i will try to see how ugly an empty list will make this; i expect a couple parts will get more funky than they currently are — we'll see.

was not horrible; pushed updates.

Reply:
folded those changes into the "replace datasets..." commit, thanks.